roles/openshift_master/files/.gitignore

```diff
@@ -0,0 +1 @@
+/openshift
```

Comment:
if we do this with explicit paths instead of the cp stuff, this shouldn't be needed (same with the other .gitignores with the same entry)